ingest_api/runtime/Dockerfile

```
@@ -1,9 +1,12 @@
 FROM public.ecr.aws/sam/build-python3.9:latest
 
+ENV VEDA_DB_PGSTAC_VERSION=0.7.4
```

```suggestion
ARG PGSTAC_VERSION
RUN echo "Using PGSTAC Version ${PGSTAC_VERSION}"
```
I think this should be coming directly from the build environment so we can try removing this line entirely. If not we may need add pgstac version to the via the ingest_api/infrastructure.

I think it will work like this but it may need a little trial and error:

1. For the config can mostly follow this example from the [database construct](https://github.com/NASA-IMPACT/veda-backend/blob/develop/database/infrastructure/config.py#L23-L26). 
2. But we'll need to call it db_pgstac_version in ingest_api/infrastructure/config.py if we want to pick up the same environment variable in the ingest api scope (some configs are prefixed with their scope)
3. To get the config into the ingestor, we'll can add it as a build arg for the [ingestor construct](https://github.com/NASA-IMPACT/veda-backend/blob/develop/ingest_api/infrastructure/construct.py#L316) that uses pypgstac. There's a [build arg](https://github.com/NASA-IMPACT/veda-backend/blob/develop/database/infrastructure/construct.py#L55) example in the database construct.